README.md

```diff
@@ -213,4 +213,23 @@ if err != nil {
     fmt.Println(err)
 }
 fmt.Println(keys)
+```
+
+### Using Search functionality in list Keys API
```

Comment:
Can you add a comment with the link to API docs that explain how the search query is getting formed.